src/controllers/controller.line.js

```diff
@@ -112,9 +116,22 @@ module.exports = DatasetController.extend({
 		var value = dataset.data[index];
 		var yScale = me.getScaleForId(meta.yAxisID);
 		var xScale = me.getScaleForId(meta.xAxisID);
+		var POINT_ELEMENT_OPTIONS = {
```

Comment:
Could these live in element.point instead, of are there conflicts in naming? Not sure they should, just thinking out loud.

Reply:
I'm not sure I understand what the suggestion is.

To make sure we're on the same page, this is a mapping from element option name to dataset option name

Reply:
I think it should stay in this file as long as we don't need it elsewhere

Reply:
I was asking if those option names are always the same. (seems to be for line and radar, for bubble they are without the `point` prefix), and could therefore be defined only once in the place they are effectively used in (elements.point.js).

So the answer is **no** (or _not yet_).

Ideally this map would come from the `defaults` for the element (and prefix would be elements name or the options would be scoped (`point.borderColor` insteand of `pointBorderColor`)). Maybe in v3.